client/app/billing/autoRenew/BillingService.js

```diff
@@ -0,0 +1,113 @@
+import _ from 'lodash';
+
+export default class BillingService {
```

Comment:
Should be in ˋBillingService.class.js` ☀️

Reply:
Should be in holidays ☀️